demo/simple/mzcompose.yml

```diff
@@ -7,12 +7,17 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
+x-port-mappings:
+  - &materialized 6875:6875
```

Comment:
FWIW I think this pattern makes things more confusing for users. Relies on a lot of unfamiliar YAML syntax. I think we'd be better off just hardcoding 6875:6875 around.

This is a meta-point, so don't worry about addressing here—I know it's a pattern that's proliferated throughout our mzcompose.yml files.

Reply:
(It also might be controversial, so definitely don't do anything about it here.)

Reply:
Yeah I'm not real into this syntax - I find it confusing, and we have some demos where the port ends up hardcoded in another location that doesn't use the yaml anchor.

We've also got https://github.com/MaterializeInc/materialize/blob/main/demo/trx/docker-compose.yaml which defines all the anchors... then doesn't use them.